docs/guides/nrfconnect_android_commissioning.md

```diff
@@ -60,31 +61,42 @@ applications:
 You need the following hardware and software for commissioning the nRF Connect
 accessory using Android CHIPTool:
 
--   Two nRF52840 DK (PCA10056)
+-   1x smartphone with Android 8+
+-   1x Wi-Fi Access Point supporting IPv6 (without the IPv6 Router Advertisement
```

Comment:
This is needed for both Thread and Wi-Fi?

Reply:
Yes both of them. Raspberry Pi as Thread Border Router uses this Wi-Fi access point.